tests/components/myuplink/test_number.py

```diff
@@ -101,5 +98,5 @@ async def test_entity_registry_smo20(
 ) -> None:
     """Test that the entities are registered in the entity registry."""
 
-    entry = entity_registry.async_get("number.f730_cu_3x400v_change_in_curve")
+    entry = entity_registry.async_get("number.gotham_city_change_in_curve")
```

Comment:
Why did the entity id change?

Reply:
I think because this smo20 got merged and 2 hours later https://github.com/home-assistant/core/pull/111502 got merged which fixes naming. I think the latter wasn't rebased, which caused this